src/cascadia/TerminalApp/ITab.idl

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import "Command.idl";
+
+namespace TerminalApp
+{
+    interface ITab
+    {
+        String Title { get; };
+        Windows.UI.Xaml.Controls.IconSource IconSource { get; };
+        Command SwitchToTabCommand { get; };
+
+        Microsoft.UI.Xaml.Controls.TabViewItem GetTabViewItem();
+        Windows.UI.Xaml.UIElement GetTabContent();
+        String GetActiveTitle();

```

Comment:
may wanna make this a property!

Reply:
I've got a question around this one - would you say `TerminalTab` should consider `Title()` and `GetActiveTitle()` to return the same thing? When would we _not_ want the "active title"?

Reply:
That’s a Q for @zadjii-msft! IDK why we would have separate titles